packages/experimental/Link/src/LinkTokens.ts

```diff
@@ -0,0 +1,66 @@
+import { Theme } from '@fluentui-react-native/framework';
+import { TokenSettings } from '@fluentui-react-native/use-styling';
+import { LinkTokens } from './Link.types';
+
+export const defaultLinkTokens: TokenSettings<LinkTokens, Theme> = (t: Theme) =>
+  ({
+    color: t.colors.link,
+    alignSelf: 'flex-start',
+    variant: 'body1',
+    inline: {
+      textDecorationLine: 'underline',
+    },
+    disabled: {
+      color: t.colors.disabledBodyText,
```

Comment:
I think this is an old token. We should be using only Fluent2 alias tokens for colors for the new components.

Reply:
This is the main blocker for this PR

Reply:
Updated to use the new colors